enterprise/frontend/src/embedding-sdk/types/theme/index.ts

```diff
@@ -180,6 +183,21 @@ export type MetabaseComponentTheme = {
     };
   };
 
+  /** Tooltip */
+  tooltip?: {
+    /** Tooltip text color. */
+    textColor?: string;
+
+    /** Tooltip text color for muted rows. */
+    secondaryTextColor?: string;
```

Comment:
In the PR description screenshot, this is _May 2024_, _22.72_ and _+67.74%_

Reply:
It's not really for only muted rows then. This secondary text color could be used for percentage change or the date as well. I'd add examples to the comment, or just remove it to avoid confusion.

The same thing could be said for the readme as well.